core/controllers/reader_test.py

```diff
@@ -205,6 +248,103 @@ def test_huge_answers(self):
+        # self.EXP_STATE = exp_services.get_exploration_by_id(exploration_id)
```

Comment:
Was trying to just get the exploration by id, but that didn't work, so this is an intermediate hack. Any suggestions?